src/components/ActionControls/ActionControls.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import getProp from 'lodash/get';
+
+import PlainButton from 'box-react-ui/lib/components/plain-button';
+import IconHighlightAnnotation from 'box-react-ui/lib/icons/annotations/IconHighlightAnnotation';
+import IconHighlightCommentAnnotation from 'box-react-ui/lib/icons/annotations/IconHighlightCommentAnnotation';
+import IconClose from 'box-react-ui/lib/icons/general/IconClose';
+import CommentText from '../../../third-party/components/Comment/CommentText';
+
+import { TYPES } from '../../constants';
+import { isHighlightAnnotation } from '../../util';
+import Internationalize from '../Internationalize';
+
+import messages from './messages';
+
+import './ActionControls.scss';
+
+type Props = {
+    id: string,
+    type: Annotations,
```

Comment:
help me understand, is type really of type Annotations?

Reply:
whoops! that should be string!